Modelica/Mechanics/Translational.mo

```diff
@@ -1861,14 +1861,14 @@ force law in a target system between two masses.
         end for;
         annotation (Documentation(info="<html>
 <p>
-Returns a table with the friction characteristic table[nTable,2] = [0, f1; ....; v_max, fn], where the first
-column is the velocity v in the range 0..v_max and the second column is the friction force
+Returns a table with the friction characteristic table[nTable,2] = [0,&nbsp;f1; &hellip;; v_max,&nbsp;fn], where the first
```

Comment:
Why no nbsp here before and after the ellipsis?

Reply:
Well then this should be:  `table[nTable,2]&nbsp;=&nbsp;[0,&nbsp;f1;&nbsp;&hellip;;&nbsp;v_max,&nbsp;fn]` 
But then the non-breaking text would be too long to my feeling.

Reply:
I have changed this.